compiler/luci-interpreter/src/loader/KernelBuilder.test.cpp

```diff
@@ -202,6 +203,19 @@ TEST_F(KernelBuilderTest, AveragePool2D)
   EXPECT_THAT(kernel->params().activation, Eq(op->fusedActivationFunction()));
 }
 
+TEST_F(KernelBuilderTest, Cast)
+{
+  auto *input = createInputNode();
+
+  auto *op = createNode<luci::CircleCast>();
+  op->x(input);
+
+  auto kernel = buildKernel<kernels::Cast>(op);
+
+  checkTensor(kernel->input(), input);
```

Comment:
@Bronnikoff @seanshpark New SVACE critical critical occurs here. (Can we add link to SVACE?)

Reply:
> Can we add link to SVACE?

Not sure ... @lemmaa , is it OK?

Reply:
I think adding
```cxx
ASSERT_THAT(kernel, NotNull());
```
may solve the problem... I'll post a PR